helm/charts/hydra/README.md

```diff
@@ -8,131 +8,132 @@ A Helm chart for deploying ORY Hydra in Kubernetes
 
 ## Maintainers
 
-| Name | Email | Url |
-| ---- | ------ | --- |
+| Name     | Email     | Url                 |
```

Comment:
Those files are auto-generated by helm-docs plugin from our values.yaml. The `make format` and the plugin will override each other all the time 😞